code/test/com/sixsq/nuvla/server/resources/deployment_set_lifecycle_test.clj

```diff
@@ -1638,6 +1639,49 @@
                                        :resources    {:cpu   {:capacity 3}
                                                       :ram   {:capacity 1024}}})))))))))
 
+(deftest lifecycle-auto-update
```

Comment:
I know it's still a wip but my suggestion is to add a call to auto-update action

Reply:
Indeed, will do, tnx.